### PR TITLE
add convert module

### PR DESCRIPTION
**Pull request by sina-devel**
added convert module (digit converts) and removed unused libs

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -7,5 +7,3 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-lazy_static = "1.4.0"
```
Comment (robatipoor) [inline Cargo.toml:10]: please do not remove dependencies, we need them in the future

File: src/convert/mod.rs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+pub enum Lang {
+    Fa,
+    En,
+}
+
+fn digits_convert(src: &str, from: Lang, to: Lang) -> String {
+    src.chars()
+        .map(|v| {
+            let index = DIGITS[from as usize].iter().position(|&r| r == v);
+            if index.is_some() {
```
Comment (robatipoor) [inline src/convert/mod.rs:16]: please use Instead `if index.is_some() ` use `if let Some(i) = DIGITS...`

File: src/convert/mod.rs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                return DIGITS[to as usize][index.unwrap()];
+            }
+            return v;
+        })
+        .collect::<String>()
+}
+
+pub fn digits_tofa(src: &str) -> String {
```
Comment (robatipoor) [inline src/convert/mod.rs:24]: please use snake case naming style like digits_to_fa or digits_to_farsi

File: src/convert/mod.rs [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    digits_convert(src, Lang::En, Lang::Fa)
+}
+
+pub fn digits_toen(src: &str) -> String {
```
Comment (robatipoor) [inline src/convert/mod.rs:28]: also here

**Review by robatipoor — COMMENTED** [review]
very thank you

File: Cargo.toml (hunk 1 above)
Comment (sina-devel) [reply Cargo.toml:10]: ok

File: src/convert/mod.rs (hunk 4 above)
Comment (sina-devel) [reply src/convert/mod.rs]: all fixed

**Comment by robatipoor** [comment]
thanks

File: src/convert/mod.rs [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+pub enum Lang {
+    Fa,
+    En,
+}
+
+fn digits_convert(src: &str, from: Lang, to: Lang) -> String {
+    src.chars()
+        .map(|v| {
+            let index = DIGITS[from as usize].iter().position(|&r| r == v);
+            if let Some(i) = index {
```
Comment (omid) [inline src/convert/mod.rs:16]: @sina-devel We can have a match instead of this, then you can omit the return

**Pull request merged**